recipe/bld.bat

```diff
@@ -1,4 +1,4 @@
-%PYTHON% setup.py install --curl-dir=%LIBRARY_PREFIX% --use-libcurl-dll
+%PYTHON% setup.py install --curl-dir=%LIBRARY_PREFIX% --use-libcurl-dll --with-schannel
```

Comment:
conda-forge moved to `pip install` instead of `setup.py install`, https://github.com/conda-forge/pycurl-feedstock/blob/main/recipe/bld.bat